plugins/obs-filters/sdr-on-hdr-filter.c

```diff
@@ -0,0 +1,172 @@
+#include <obs-module.h>
+
+enum sdr_on_hdr_gamma {
+	GAMMA_SRGB,
+	GAMMA_POWER,
+};
```

Comment:
We usually now add an "invalid" sentinel value in first place to enums to ensure that values that are zero-initialised by default cannot be a valid enumeration value and that the enumeration value needs to be explicitly initialised/specified.